packages/zoe/test/unitTests/contracts/helpers/test-bondingCurves.js

```diff
@@ -2,7 +2,8 @@
 import { test } from 'tape-promise/tape';
 import harden from '@agoric/harden';
 
-import { makeConstProductBC } from '../../../../src/contracts/helpers/bondingCurves';
+// Should be imported from '@agoric/zoe-contractSupport'
```

Comment:
These tests will probably move with the code, right? So in that case, they will always be local imports and we don't need this comment.

Reply:
@Chris-Hibbert I just realized, the tests are supposed to have the same folder structure as the src code. Can we move these tests out into folders under unitTests?